documentation/docs/30-advanced/60-images.md

```diff
@@ -0,0 +1,149 @@
+
+Since the `enhanced:img` tag will be replaced with a `picture` tag, you should refer to it as such within your `style` tag. Generally we recommend styling the `img` child of the `picture` tag rather than the `picture` tag itself to make it easier to switch between various solutions such as a dynamic CDN-based solution should you ever need to do so.
+
+### Dynamically choosing an image
+
+You can also manually import an image and then pass it to an `enhanced:img` tag. This is useful when you have a collection of static images and would like to dynamically choose one. You can create a collection of images manually [as we do on the homepage showcase](https://github.com/sveltejs/kit/blob/master/sites/kit.svelte.dev/src/routes/home/Showcase.svelte). In this case you will need to update both the `import` statement and `img` tag as shown below to indicate you'd like process them.
+
+```svelte
+<script>
+	import { MyImage } from './path/to/your/image.jpg?enhancedImg';
+</script>
+
+<enhanced:img src={MyImage} alt="An alt text" />
+```
+
+You can also use [Vite's `import.meta.glob`](https://vitejs.dev/guide/features.html#glob-import). Note that you will have to specify `enhancedImg` via a [custom query](https://vitejs.dev/guide/features.html#custom-queries):
+
+```js
+const pictures = import.meta.glob(
+	'/path/to/assets/*.{avif,gif,heif,jpeg,jpg,png,tiff,webp}',
+	{
+		query: {
+			enhancedImg: true
+		}
+	}
+);
+```
+
+### Intrinsic Dimensions
+
+`width` and `height` are optional as they can be inferred from the source image and will be automatically added when the `<enhanced:img>` tag is preprocessed. These attributes are added because the browser can reserve the correct amount of space when it knows image dimensions which prevents layout shift. If you'd like to use a different `width` and `height` you can style the image with CSS. Because the preprocessor adds a `width` and `height` for you, if you'd like one of the dimensions to be automatically calculated then you will need to specify that:
+
+```svelte
+<style>
+	.hero-image img {
+		width: var(--size);
+		height: auto;
+	}
+</style>
+```
+
+### `srcset` and `sizes`
+
+You may be wondering how to create something that looks like:
+
+```html
+<img
+  srcset="image-small.png 160w, image-medium.png 400w, image-large.png 600w"
+  sizes="(min-width: 60rem) 80vw, (min-width: 40rem) 90vw, 100vw"
+  />
+```
+
+In this example, it would be tedious to have to manually create three versions of your image. Instead, you can specify the widths as a query parameter and we'll generate the `srcset` for you.
+
+```svelte
+<enhanced:img
+  src="./image.png?w=160,400,600"
+  sizes="(min-width: 60rem) 80vw, (min-width: 40rem) 90vw, 100vw"
```

Comment:
where did we land on defaulting this to `100vw`?

Reply:
It's a good question. At the very least we should probably simplify this example or provide a simpler one alongside it.

I'll take some time next week to think about and test it

Reply:
I don't think we should default to `100vw` (provided some more detail on Discord), but I do agree this looked a bit gnarly, so updated this section with a more realistic example